docs/build/cicd/ado.adoc

```diff
@@ -0,0 +1,64 @@
+:src_docs_path: docs/docker-definitions
+:dst_docs_path: src/definitions
+
+=== Azure DevOps
+
+A pipeline has been configured for Azure DevOps, which calls `taskctl`, and runs the tasks as described previously.
```

Comment:
Previously where? Should we link to where you mean?

Reply:
This is because the file is part of a larger document which is stitched together to make one bigger document.
I have modified the sentence